Api/Entities/OAuthToken.cs

```diff
@@ -17,12 +19,24 @@ public class OAutToken
         /// <summary>
         /// Gets a DateTime with when this token expires
         /// </summary>
-        public DateTime ExpiresAt { get; private set; }
+        public DateTime ExpiresAt
```

Comment:
Hi @5E7EN thanks for your contribution, would you mind following the same pattern as the RingApi project to keep it consistent please?
https://github.com/KoenZomers/RingApi/blob/master/Api/Entities/OAuthToken.cs#L28